### ISSUE TITLE
add Bumpers

### ISSUE THREAD

**Issue by ListLostLisp** [mopenstein/raspberry_pi_tv_station#1]
I haven't used this yet but noticed there wasn't any mention of bumpers in the code. Would you be interested in adding it? I was thinking an option to play a different bumper for a certain time/show. and short bumpers in-between episodes. 

It could maybe be auto generated to automatically show the time and what episode is next. could use flash, that's what was mainly used in actual tv iirc

Sorry for the big ask, this project is the only open source playout software I've found that does almost everything including support for fade to blacks.
 Even just support to play a certain bumper file before or after the commercials would be nice.

**Comment by mopenstein** [comment]
At one point I started to add bumpers but I ran out of interest in it because I went a more customizable direction. 

It would certainly be possible to add code to support bumpers that all ready exist and they could be programmed using the settings file. However, it adds a layer of complexity when auto generating commercials to fill time to the exact half/full hour.

Auto generating content is outside the scope of this project as I see it.
